### DISCUSSION TITLE
It is possible to connect zwave-js Home Assistant remotely?

### DISCUSSION THREAD

**Discussion by AlfaBravoX** [zwave-js/zwave-js-ui#3281] [Q&A] [answered]
Hi community,

I run two installations of zwave-js-ui in two different buildings on 2 different LANs 192.168.2.0 and 192.168.4.0

Home assistant is installed first LAN with IP 192.168.2.3 and have zwave-js-ui configured as Addon.

Now I am wondering if zwave-js-ui with 192.168.2.102 (docker) can be integrated to Home Assistant too and Home assistant can handle both zwave-js-ui  instance.

Of course both LANs are routable and they can see each other normally. 

Thanks

**Comment by robertsLando** [comment]
I think this should be possible but I let @kpine answer as I'm not an Home Assistant expert

**Comment by kpine** [comment, marked as answer]
Yes, you just add a second instance of the integration, configured to connect to the second ZUI.

**Reply by AlfaBravoX** [reply]
Could you please advise how to add second instance of the integration that have ZJS UI that is running on external IP from range of 192.168.2.x ? I am able to see just to add second instance connected to again local serial port /dev/ttyAMA0

**Reply by kpine** [reply]
Settings -> Devices & Services -> Add Integration. Search for Z-Wave.

Also read the ZUI docs. https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-official

**Reply by AlfaBravoX** [reply]
Thanks, I was trying this one before, but still.. I am only getting this dialog not allowing me to add external ZUI server
<img width="522" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/11148591/c7035f75-bdb5-49d3-9b9a-940b183a8146">

**Reply by kpine** [reply]
You're not supposed to check the Supervisor option.

https://www.home-assistant.io/integrations/zwave_js/#installing-and-configuring-the-z-wave-integration-in-home-assistant

**Reply by AlfaBravoX** [reply]
oh, my bad. works like a charm, many thanks!

<img width="565" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/11148591/0694af36-9cff-4dc7-a837-7dab1cdade72">
